### DISCUSSION TITLE
Fatal:Unable to access https://github.com/google/googletest /": Failed to connect to github.com port 443:Time out

### DISCUSSION THREAD

**Discussion by selvavg** [google/googletest#3798] [Community Help]
Hi Friends,

Good morning , while run the plan from our Bamboo , we got below error concurrently , is it happend due to https://github.com/google/googletest / page get down short time or any blocker . plz support.

**Error**:
Fatal:Unable to access https://github.com/google/googletest /": Failed to connect to github.com port 443:Time out .
Failed to clone 'external/googletest' a second time, aborting

Thanks
Selva.V

**Comment by Hussain1811** [comment]
Please check your proxy settings
